### PR TITLE
[zh-cn]Update service-v1.md

### PR DESCRIPTION
**Pull request by suning0**

### CODE REVIEW

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md [hunk 1]
```diff
@@ -363,34 +363,36 @@ ServiceSpec 描述用户在服务上创建的属性。
...
+  healthCheckNodePort specifies the healthcheck nodePort for the service. This only applies when type is set to LoadBalancer and externalTrafficPolicy is set to Local. If a value is specified, is in-range, and is not in use, it will be used.  If not specified, a value will be automatically allocated.  External systems (e.g. load-balancers) can use this port to determine if a given node holds endpoints for this service or not.  If this field is specified when creating a Service which does not need it, creation will fail. This field will be wiped when updating a Service to no longer need it (e.g. changing type). This field cannot be updated once set.
   -->
   healthCheckNodePort 指定 Service 的健康检查节点端口。
   仅适用于 type 为 LoadBalancer 且 externalTrafficPolicy 设置为 Local 的情况。
   如果为此字段设定了一个值，该值在合法范围内且没有被使用，则使用所指定的值。
   如果未设置此字段，则自动分配字段值。外部系统（例如负载平衡器）可以使用此端口来确定给定节点是否拥有此服务的端点。
-  在创建不需要 healthCheckNodePort 的 Service 时指定了此字段，则 Service 创建会失败。
+  在创建不需要 healthCheckNodePort 的 Service 时指定了此字段，则 Service 创建会失败。该字段一旦设置就无法更新。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md:392]: dup with line 394?
```suggestion
  在创建不需要 healthCheckNodePort 的 Service 时指定了此字段，则 Service 创建会失败。
```

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md [hunk 2]
```diff
@@ -363,34 +363,36 @@ ServiceSpec 描述用户在服务上创建的属性。
...
   <!-- 
-  InternalTrafficPolicy specifies if the cluster internal traffic should be routed to all endpoints or node-local endpoints only. "Cluster" routes internal traffic to a Service to all endpoints. "Local" routes traffic to node-local endpoints only, traffic is dropped if no node-local endpoints are ready. The default value is "Cluster". 
+  InternalTrafficPolicy describes how nodes distribute service traffic they receive on the ClusterIP. If set to "Local", the proxy will assume that pods only want to talk to endpoints of the service on the same node as the pod, dropping the traffic if there are no local endpoints. The default value, "Cluster", uses the standard behavior of routing to all endpoints evenly (possibly modified by topology and other features).
   -->
-  internalTrafficPolicy 指定是将集群内部流量路由到所有端点还是仅路由到节点本地的端点。
-  字段值 “Cluster” 将 Service 的内部流量路由到所有端点。
-  字段值 ”Local” 意味着仅将流量路由到节点本地的端点；如果节点本地端点未准备好，则丢弃流量。
-  默认值为 “Cluster”。
+  InternalTrafficPolicy 描述了节点如何分发它们在 ClusterIP 上接收到的服务流量。 
+  如果设置为“本地”，代理将假定 pod 只想与与 pod 位于同一节点上的服务端点通信，如果没有本地端点则丢弃流量。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md:380]:
```suggestion
  如果设置为“Local”，代理将假定 Pod 只想与同一节点上的 Pod 的服务端点通信，如果没有本地端点则丢弃流量。 
```

File: content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md [hunk 3]
```diff
@@ -363,34 +363,36 @@ ServiceSpec 描述用户在服务上创建的属性。
...
   -->
-  externalTrafficPolicy 表示此 Service 是否希望将外部流量路由到节点本地或集群范围的 Endpoint。
-  字段值 “Local” 保留客户端 IP 并可避免 LoadBalancer 和 Nodeport 类型 Service 的第二跳，但存在潜在流量传播不平衡的风险。
-  字段值 “Cluster” 则会掩盖客户端源 IP，可能会导致第二次跳转到另一个节点，但整体流量负载分布较好。
+  externalTrafficPolicy 描述了节点如何在 Service 的“面向外部”地址（NodePorts、ExternalIPs 和 LoadBalancer IPs）之一上分配它们接收到的服务流量。 
+  如果设置为“Local”，代理将以一种假定外部负载平衡器负责平衡节点之间的服务流量的方式配置 Service ，因此每个节点将仅向 Service 的节点本地端点传送流量, 无需伪装客户端源 IP。
+  （错误地发送到没有端点的节点的流量将被丢弃。）
+  默认值“Cluster”使用标准行为均匀路由到所有端点（可能由拓扑和其他功能修改）。
+  请注意，从集群内发送到外部 IP 或 LoadBalancer IP 的流量将始终获得“Cluster”语义，但从集群内发送到 NodePort 的客户端在选择节点时可能需要考虑流量策略。
```
Comment (howieyuen) [inline content/zh-cn/docs/reference/kubernetes-api/service-resources/service-v1.md:372]:
```suggestion
  请注意，从集群内发送到外部 IP 或 LoadBalancer IP 的流量将始终获得 “Cluster” 语义，但从集群内发送到 NodePort 的客户端在选择节点时可能需要考虑流量策略。
```

**Comment by tengqm** [comment]
@suning0 Still interested in working on this?

**Comment by tengqm** [comment]
@suning0 Still interested in working on this PR?

**Comment by Sea-n** [comment]
Hi @suning0, could you address [the above issue](https://github.com/kubernetes/website/pull/38134#pullrequestreview-1196831501)?
/lgtm cancel

**Pull request closed without merging**